examples/evaluation/rephrase_prompts.ipynb

```diff
@@ -0,0 +1,805 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Rephrasing prompts using `prompto`\n",
+    "\n",
+    "We illustrate how we can use `prompto` to rephrase prompts. This is useful if you first want to generate a more diverse set of prompts and then use them to generate a more diverse set of completions."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from prompto.settings import Settings\n",
+    "from prompto.experiment import Experiment\n",
+    "from prompto.rephrasal import Rephraser, load_rephrase_folder\n",
+    "from dotenv import load_dotenv\n",
+    "import json\n",
+    "import os"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "When using `prompto` to query models from the Ollama API, lines in our experiment `.jsonl` files must have `\"api\": \"ollama\"` in the prompt dict. \n",
+    "\n",
+    "## Setting up Ollama locally\n",
+    "\n",
+    "In this notebook, we assume that you have a local instance of the Ollama API running. For installing Ollama, please refer to the [Ollama documentation](https://ollama.com/). Once you have it installed and have it running, e.g. with `ollama serve` in the terminal, you can proceed with the following steps.\n",
+    "\n",
+    "\n",
+    "By default, the address and port that Ollama uses when running is `localhost:11434`. When developing this notebook, we were running Ollama locally so we set the `OLLAMA_API_ENDPOINT` to `http://localhost:11434`. If you are running the server at a different address or port, you can specify with the `OLLAMA_API_ENDPOINT` environment variable accordingly as described below.\n",
+    "\n",
+    "### Downloading models\n",
+    "\n",
+    "In this notebook and our example experiment file ([example_file.jsonl](https://github.com/alan-turing-institute/prompto/tree/main/examples/evaluation/example_file.jsonl)), we have set to query from Gemma2 models and we will first rephrase these prompts using Llama 3.2 - note that Ollama defaults to the smaller versions of these (8B, 2B). You can download these models using the following commands in the terminal:\n",
```

Comment:
Highlight Llama 3.2 because people might miss it if they read fast, like i was doing :D